src/apps/Agreement/ConfigurationChecklist.js

```diff
@@ -0,0 +1,209 @@
+import React, { useEffect, useState } from 'react'
+import {
+  Box,
+  useLayout,
+  GU,
+  useTheme,
+  ProgressBar,
+  textStyle,
+  Button,
+  springs,
+} from '@aragon/ui'
+import PropTypes from 'prop-types'
+import checklistCompleteGraphic from './assets/checklist-complete.png'
+import { Transition, animated } from 'react-spring'
+
+function countProgress(items) {
+  return items.filter(([_, checked]) => checked).length
+}
+
+const AnimatedDiv = animated.div
+
+function ConfigurationChecklist({ items, onClose }) {
+  const [progress, setProgress] = useState(countProgress(items))
+  const theme = useTheme()
+
+  const barProgress = (1 / items.length) * progress
+  const checklistComplete = items.length === progress
+
+  useEffect(() => {
+    setProgress(countProgress(items))
+  }, [items])
+
+  return (
+    <Box heading="Configuration checklist" padding={0}>
+      <Transition
+        native
+        items={checklistComplete}
+        from={{
+          opacity: 0,
+          transform: 'scale(0.7)',
+        }}
+        enter={{
+          opacity: 1,
+          transform: 'scale(1)',
+        }}
+        config={springs.smooth}
+      >
+        {show =>
+          show &&
+          (props => (
+            <BoxInner>
+              <AnimatedDiv
+                style={props}
+                css={`
+                  display: flex;
+                  flex-direction: column;
+                  align-items: center;
+                `}
+              >
+                <h2
+                  css={`
+                    text-align: center;
+                    margin-bottom: ${2 * GU}px;
+                  `}
+                >
+                  All done! Congratulations!
+                </h2>
+                <img
+                  src={checklistCompleteGraphic}
+                  css={`
+                    height: 130px;
+                  `}
+                />
+              </AnimatedDiv>
+            </BoxInner>
+          ))
+        }
+      </Transition>
+      {!checklistComplete &&
+        items.map(([label, checked]) => (
+          <BoxInner key={label}>
+            <ChecklistItem label={label} checked={checked} />
+          </BoxInner>
+        ))}
+
+      <BoxInner>
+        <div
+          css={`
+            padding-bottom: ${1 * GU}px;
+          `}
+        >
+          <label
+            css={`
+              ${textStyle('label2')}
```

Comment:
Nitpick, but we usually put `textStyle()` last as a convention (and then any overrides after)